prom2teams/message/parser.py

```diff
@@ -27,15 +26,12 @@ def check_fields(json_alerts_attr, json_alerts_labels_attr, json_alerts_annotati
                 alert_fields[alert_field_key] = 'unknown'
     return alert_fields
 
-def parse(json_str):
```

Comment:
Now that you are using flask-restplus, it's easier to user http://flask-restplus.readthedocs.io/en/stable/parsing.html
Also is a win to use http://flask-restplus.readthedocs.io/en/stable/swagger.html